### PR TITLE
Improve docs for config file in components

### PR DESCRIPTION
**Pull request by zongUMR**
Improve: https://github.com/serverless/roadmap-tencent/issues/775
Task: https://app.asana.com/0/1200011502754281/1200025989019509

Components:
- [x] Next.js: https://github.com/serverless-components/tencent-examples/pull/66/commits/b1e508dc67d51252356791cbc0fba876108ff812
- [x] Nuxt.js: https://github.com/serverless-components/tencent-examples/pull/66/commits/dc60fed17853d1746a25b84dc264a41021b60b81
- [x] react-starter: https://github.com/serverless-components/tencent-examples/pull/66/commits/508f9a5ee123a570225a20744159514f14c20f2f
- [x] Fullstack: https://github.com/serverless-components/tencent-examples/pull/66/commits/a5275cee9414b8071df23423e08b4150458a87df
- [x] Eggjs: https://github.com/serverless-components/tencent-examples/pull/66/commits/8df6a94af901c581cb8a3614fb43dde3a4c8df19
- [x] SCF-nodejs: https://github.com/serverless-components/tencent-examples/pull/66/commits/d691215998b73f7e12e7ac45dc61ce3aaf190b79
- [x] Express

### CODE REVIEW

File: fullstack/src/frontend/serverless.yml [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
-    dist: ./dist
-    envPath: ./
-    index: index.html
-    error: index.html
-  env:
+  region: ${env:REGION} # 云函数所在区域
+  bucketName: fullstack-serverless-frontend # Bucket 名称。 不允许大写字母。如果你不加 AppId 后缀，则默认自动会为你加上。
+  protocol: https # 请求协议。https 或 http
+  src: # 该项目的代码信息，参数参考执行目录
+    src: ./ # 你构建的项目代码目录。
```
Comment (yugasun) [inline fullstack/src/frontend/serverless.yml:26]: If config `hook` and  `dist`, `src` is no need.

File: react-starter/src/serverless.yml [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+# 更多内容请查看：https://github.com/serverless-components/tencent-website/blob/master/docs/configure.md
 inputs:
-  src:
-    src: ./
-    hook: npm run build
-    dist: ./dist
-  bucketName: my-react-starter
-  protocol: https
+  src: #该项目的代码信息，参数参考执行目录
+    src: ./ # 你构建的项目代码目录
```
Comment (yugasun) [inline react-starter/src/serverless.yml:22]: If config `hook` and `dist`, `src` is no need.

**Review by yugasun — CHANGES_REQUESTED** [review]

**Review by timqian — APPROVED** [review]
LGTM

**Comment by ole3021** [comment]
Update the doc with a commit

* reorder the basic info
* Update optional value list
* fix some comments

**Comment by ole3021** [comment]
* 统一注释内容和样式
* 添加引用链接
* 添加express 注释帮助
* 精简注释内容
* 移除多余注释内容

**Comment by zongUMR** [comment]
>     * 统一注释内容和样式
> 
>     * 添加引用链接
> 
>     * 添加express 注释帮助
> 
>     * 精简注释内容
> 
>     * 移除多余注释内容

` express-starter/src/serverless.yml ` has conflicts on your this commit

**Comment by timqian** [comment]
express-starter 的 yml 没修完全，inputs 等内容有重复 @ole3021

**Comment by ole3021** [comment]
Have checked and tested all config files.

**Review by ole3021 — APPROVED** [review]

**Review by timqian — APPROVED** [review]

**Pull request merged**